examples/ui/size_constraints.rs

```diff
@@ -358,13 +350,14 @@ fn update_buttons(
 }
 
 fn update_radio_buttons_colors(
-    mut event_reader: EventReader<ButtonActivatedEvent>,
+    mut click_events: EventReader<Click>,
     button_query: Query<(Entity, &Constraint, &Interaction)>,
     mut color_query: Query<&mut BackgroundColor>,
     mut text_query: Query<&mut Text>,
     children_query: Query<&Children>,
 ) {
-    for &ButtonActivatedEvent(button_id) in event_reader.read() {
+    for event in click_events.read() {
+        let button_id = event.0;
```

Comment:
```suggestion
    for &Click(button_id) in click_events.read() {
```
Would it work to keep the same destructuring as before?